src/views/Home.jsx

```diff
@@ -1,17 +1,26 @@
 import './Home.css';
 import toast, { Toaster } from 'react-hot-toast';
 import { generateToken } from '@the-collab-lab/shopping-list-utils';
-import { useEffect } from 'react';
+import { useEffect, useState } from 'react';
 import { useNavigate } from 'react-router-dom';
 import { createNewList } from '../api/firebase';
+import { queryForCollection } from '../api/firebase';
 
 /** Home component that redirects a user to the List view if there is already a list created.
- * If the user doesn't already have a list, a user can create one to be saved to Firestore and be redirected to the List view. */
+ * If the user doesn't already have a list, a user can create a list or join one to be saved to Firestore and be redirected to the List view. */
 
 export function Home({ setListToken }) {
-	// Declaring navigate for view redirection and errorNotify for error handling.
+	//Declare useState variable for user shared token input
+	const [joinToken, setJoinToken] = useState('');
+
+	// Declare navigate for view redirection
 	const navigate = useNavigate();
-	const errorNotify = () => toast.error('Error creating new shopping list ');
+
+	// Declare error notifying functions using react-hot-toast.
+	const listCreateError = () => toast.error('Error creating new shopping list');
+	const listJoinError = () => toast.error('This shopping list does not exist');
+	const blankError = () =>
+		toast.error('Error: Shopping list token cannot be blank');
```

Comment:
I love seeing all the toasts!! They look great when I run the app locally, too. Also like that the names are very intuitive for what they're used for!

Reply:
Yes, the organization and comments are all very helpful.  A toast to the toasts!

Reply:
came here to say that i loved seeing ` Shopping list token cannot be blank`